### PR TITLE
Dont print extra service RR on the label

### PR DESCRIPTION
**Pull request by marcelobelli**
Correios asks to remove the RR info from the label because all PAC/Sedex is also registered.

### CODE REVIEW

File: correios/models/user.py [hunk 1]
```diff
@@ -209,6 +209,10 @@ def __init__(self, number: int, code: str, name: str) -> None:
             raise InvalidExtraServiceError("Invalid Extra Service Name {!r}".format(name))
         self.name = name
 
+        if type(display_on_label) is not bool:
```
Comment (cacarrara) [inline correios/models/user.py]: Is it really necessary to check type here? I thought declaring type in parameter definition was enough.

File: correios/renderers/pdf.py [hunk 2]
```diff
@@ -135,6 +135,9 @@ def draw(self):
         # extra services (first three)
         first_row = self.y2 - (40 * mm) - 10  # font-size=10pt
         for extra_service in self.shipping_label.extra_services:
+            if not extra_service.display_on_label:
+                continue
```
Comment (cacarrara) [inline correios/renderers/pdf.py]: Seems there is no test passing here.

File: correios/models/user.py (hunk 1 above)
Comment (marcelobelli) [reply correios/models/user.py]: @cacarrara, I was looking the [PEP 484](https://www.python.org/dev/peps/pep-0484/) and says:
"While these annotations are available at runtime through the usual ___annotations___ attribute, no type checking happens at runtime. Instead, the proposal assumes the existence of a separate off-line type checker which users can run over their source code voluntarily."

File: correios/renderers/pdf.py (hunk 2 above)
Comment (marcelobelli) [reply correios/renderers/pdf.py]: Done the test. Thanks for the tips @cacarrara

File: correios/models/user.py (hunk 1 above)
Comment (georgeyk) [reply correios/models/user.py]: so, mypy does not work here ?

Comment (marcelobelli) [reply correios/models/user.py]: Ah @georgeyk , I think I get it now. I will remove it.

**Comment by marcelobelli** [comment]
@cacarrara e @georgeyk. Feito!

File: correios/models/user.py [hunk 3]
```diff
@@ -224,7 +226,7 @@ def is_declared_value(self):
     def get(cls, number: Union['ExtraService', int]) -> 'ExtraService':
         if isinstance(number, cls):
             return number
-        return cls(number=number, **EXTRA_SERVICES[number])
+        return cls(number=number, **EXTRA_SERVICES[number])  # type: ignore
```
Comment (gfabricio) [inline correios/models/user.py]: Is this comment necessary?

Comment (marcelobelli) [reply correios/models/user.py]: Yes, it is @gfabricio. Without mypy gives an weird error saying that `EXTRA_SERVICES[number]` is not a mapping, it is an object. That's happens when I added `'display_on_label': False` on on EXTRA_SERVICES dict.

Comment (cacarrara) [reply correios/models/user.py]: Maybe because False isn't a string?

Comment (cacarrara) [reply correios/models/user.py]: Or, because not all elements are equals (has same length and attributes).

**Comment by cacarrara** [comment]
I think it's better try to fix that "type: ignore" comment before passing this PR.

File: correios/models/user.py (hunk 3 above)
Comment (marcelobelli) [reply correios/models/user.py]: @cacarrara, I can add the `'display_on_label'` in all dicts and change
this
```
@classmethod
    def get(cls, number: Union['ExtraService', int]) -> 'ExtraService':
        if isinstance(number, cls):
            return number
        return cls(number=number, **EXTRA_SERVICES[number])  # type: ignore
```
to this
```
@classmethod
    def get(cls, number: Union['ExtraService', int]) -> 'ExtraService':
        if isinstance(number, cls):
            return number
        attrs = EXTRA_SERVICES[number]  # type: Dict[str, Any]
        return cls(number=number, **attrs)
```
Do you think it is better?

Comment (cacarrara) [reply correios/models/user.py]: Well, maybe we can go with this purpose @marcelobelli

**Comment by marcelobelli** [comment]
@cacarrara feito!

**Pull request merged**